app/views/providers/applicant_employed/index.html.erb

```diff
@@ -14,6 +14,7 @@
 
     <%= form.govuk_check_boxes_fieldset :applicant,
                                         legend: { text: page_title, size: "xl", tag: "h1" },
+                                        caption: { text: t("generic.client_means_caption"), size: "l" },
```

Comment:
Discussion: the design pattern generally as both heading and caption at the same size, what are our reasons for differing from this pattern?

Reply:
Not sure, I was just following the sizing in the design/prototype

Reply:
We did the same on the proceeding loop caption-l H1-xl
Maybe we should raise this with design so we have a consistent change?

Reply:
Discussed with @naseberry and will leave in current form and raise a design ticket to review and update across the whole service